src/org/zaproxy/zap/extension/tokengen/resources/Messages.properties

```diff
@@ -37,7 +37,8 @@ tokengen.generate.button.cancel = Cancel
 tokengen.generate.button.generate = Generate
 tokengen.generate.label.name    = Name 
 tokengen.generate.label.numTokens = Number of Tokens 
-tokengen.generate.label.type    = Type 
+tokengen.generate.label.type    = Type
+tokengen.generate.label.remove.cookies = Remove Cookie Before Sending? 
```

Comment:
`Remove (Only) Cookie Under Test`? If not selected it removes all cookies, otherwise it removes just one.